packages/api/src/middleware/auth.ts

```
@@ -0,0 +1,32 @@
+declare module 'express' {
+  interface Request {
+    userId?: string
+  }
+}
+
+export function requireAuth(req: Request, _res: Response, next: NextFunction): void {
+  try {
+    const authHeader = req.headers.authorization;
+
+    if (!authHeader || !authHeader.startsWith('Bearer ')) {
+      return next(new AuthenticationError('Authorization token required'));
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)